config/crd/bases/elemental.cattle.io_machineregistrations.yaml

```diff
@@ -95,6 +95,9 @@ spec:
                             type: boolean
                           reboot:
                             type: boolean
+                          snapshotter-type:
+                            default: btrfs
```

Comment:
🤔 ok, having this as default means that also existing machine registrations will be upgraded, and so the new machines will be installed with the btrfs snapshotter, right?
What happens for the upgrades of the already installed machines? Will they keep the other kind of snapshotter or would upgrade?

Reply:
It is a bit [down the line](https://github.com/rancher/elemental-toolkit/blob/main/pkg/action/upgrade.go#L72), but during upgrade we default to the "previous" snapshotter, so this won't affect upgrades.